composer.json

```
@@ -11,23 +11,24 @@
     "type": "library",
     "license": "BSD-3",
     "homepage": "https://www.phalconphp.com",
-    "authors" : [
+    "authors": [
         {
-            "name" : "Andres Gutierrez",
-            "email" : "andres@phalconphp.com"
+            "name": "Andres Gutierrez",
+            "email": "andres@phalconphp.com"
         },
         {
-            "name" : "Mark Garrett",
-            "email" : "mark@moderndeveloperllc.com"
+            "name": "Mark Garrett",
+            "email": "mark@moderndeveloperllc.com"
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "ext-phalcon": "~3.0",
+        "php": ">=7.2",
+        "ext-phalcon": "4.0.0-beta.2",
```

```suggestion
        "ext-phalcon": "~4.0.0-beta.2",
```

No need add psr. It is not a Vokuro dependency. In fact Phalcon has a lot of dependencies. Should we add all of them here?